### PR TITLE
feat: add support for bg as true

### PR DESCRIPTION
**Pull request by rrkumarshikhar**
### Description

### CODE REVIEW

File: rapyuta_io/clients/device.py [hunk 1]
```diff
@@ -566,7 +567,44 @@ def execute_command(self, command, retry_limit=0):
...
+        """
+
+        if not deviceids or not isinstance(deviceids, list):
+            raise ValueError("Device IDs must be provided as a non-empty list.")
+        url = self._device_api_host + DEVICE_COMMAND_API_PATH + "jobid"
+        payload = {
+            "jid": jid,
+            "device_id": deviceids[0]
+        }
+        total_time_waited = 0
```
Comment (pallabpain) [inline rapyuta_io/clients/device.py:598]:
```suggestion
        time_elapsed = 0
```

File: rapyuta_io/clients/device.py [hunk 2]
```diff
@@ -566,7 +567,44 @@ def execute_command(self, command, retry_limit=0):
...
+        wait_interval = 10
+        while total_time_waited < timeout:
```
Comment (pallabpain) [inline rapyuta_io/clients/device.py:600]:
```suggestion
        while time_elapsed < timeout:
```

File: rapyuta_io/clients/device.py [hunk 3]
```diff
@@ -566,7 +567,44 @@ def execute_command(self, command, retry_limit=0):
...
+
+        if not deviceids or not isinstance(deviceids, list):
+            raise ValueError("Device IDs must be provided as a non-empty list.")
+        url = self._device_api_host + DEVICE_COMMAND_API_PATH + "jobid"
+        payload = {
+            "jid": jid,
+            "device_id": deviceids[0]
+        }
+        total_time_waited = 0
+        wait_interval = 10
```
Comment (pallabpain) [inline rapyuta_io/clients/device.py:599]: suggestion: Define this at the top of the file or accept it as a parameter in the function.
```python
def fetch_command_result(self, jid: str, deviceids: list, timeout: int, interval: int = 10):
```

**Review by pallabpain — CHANGES_REQUESTED** [review]

File: rapyuta_io/clients/device.py (hunk 1 above)
Comment (rrkumarshikhar) [reply rapyuta_io/clients/device.py]: updated this

File: rapyuta_io/clients/device.py (hunk 2 above)
Comment (rrkumarshikhar) [reply rapyuta_io/clients/device.py]: updated this as well

File: rapyuta_io/clients/device.py (hunk 3 above)
Comment (rrkumarshikhar) [reply rapyuta_io/clients/device.py]: added the interval as parameter

**Review by ankitrgadiya — APPROVED** [review]

**Review by pallabpain — APPROVED** [review]

**Review by ankitrgadiya — APPROVED** [review]

**Pull request merged**